src/wasm-lib/kcl/src/std/shell.rs

```diff
@@ -302,6 +303,7 @@ pub async fn hollow(exec_state: &mut ExecState, args: Args) -> Result<KclValue,
 /// ```
 #[stdlib {
     name = "hollow",
+    feature_tree_operation = true,
```

Comment:
TIL we had this function 😆 . Oh boy this will be a hard icon to come up with

Reply:
Yeah I noticed looking at the engine api the other day for shell deletion. I have never used hollow in CAD. Will be hard to represent for sure. Maybe a cross section icon or something